<launch>
  <master auto="start"/>
  
  <!-- Map Server -->
  <arg name="map_file" default="$(find nav_cost_test)/map/map.yaml"/>
  <node pkg="map_server" name="map_server"  type="map_server" args="$(arg map_file)" />
  
  <!-- AMCL --> 
  <include file="$(find practice)/launch/include/amcl.launch" />

  <!-- move_base -->>
  <node pkg="move_base" type="move_base" respawn="false" name="move_base" output="screen">
    <rosparam file="$(find nav_cost_test)/param/costmap_common_params.yaml" command="load" ns="global_costmap" /> 
    <rosparam file="$(find nav_cost_test)/param/costmap_common_params.yaml" command="load" ns="local_costmap" />
    <rosparam file="$(find nav_cost_test)/param/local_costmap_params.yaml" command="load" />
    <rosparam file="$(find nav_cost_test)/param/global_costmap_params.yaml" command="load" /> 
    <rosparam file="$(find nav_cost_test)/param/base_local_planner_params.yaml" command="load" />
  </node>

  <!-- rviz -->
  <node name="rviz" pkg="rviz" type="rviz" args="-d $(find robot_description)/rviz/slam.rviz" />

</launch>
